spec.html

```diff
@@ -350,9 +350,8 @@ <h1>Iterator.prototype.map ( _mapper_ )</h1>
           1. Let _closure_ be a new Abstract Closure with no parameters that captures _iterated_ and _mapper_ and performs the following steps when called:
             1. Let _counter_ be 0.
             1. Repeat,
-              1. Let _next_ be ? IteratorStep(_iterated_).
-              1. If _next_ is *false*, return *undefined*.
-              1. Let _value_ be ? IteratorValue(_next_).
+              1. Let _value_ be ? IteratorStepValue(_iterated_).
+              1. If _value_ is ~done~, return *undefined*.
```

Comment:
```suggestion
              1. If _value_ is ~DONE~, return *undefined*.
```

Reply:
No, it's spelled `~done~`.

Reply:
all spec enums are all caps now, and it's "DONE" in https://tc39.es/ecma262/#sec-iteratorstepvalue - unless you mean that's just the rendering and it's lowercase in the ecmarkup?

Reply:
That's just the rendering, yeah.